docs/subquery_network/indexers/separated-db.md

```diff
@@ -8,7 +8,19 @@ If you want to use a separated database for the Indexing service, you can follow
 
 ## Setup Database
 
-### Install PostgreSQL With Docker Compose
```

Comment:
That's not what I mean, 
* Use db from public clouds is possible but not recommended, so we either not mention it at all or put it in the last option.
* Run db locally doesn't mean it has to use docker.

Reply:
My suggestion is we shouldn't mention how to run db here, which means everything related to postgres on docker.
Instead, we just tell people if you have a db on a different machine than the one runs coordinator service, how to config so coordinator service can connect to it.